### PR TITLE
Add docs

### LINKED ISSUE #9: Write docs
Add instructions to make training faster
- Turn on fp16 if your machine supports it.

Add relevant resources to Docs page:
- [Fine-tune Whisper (large) with LoRA & BNB powerd by PEFT ⚡️](https://github.com/Vaibhavs10/fast-whisper-finetuning/blob/main/Whisper_w_PEFT.ipynb)
- [Fine-Tune Whisper For Multilingual ASR with 🤗 Transformers](https://huggingface.co/blog/fine-tune-whisper) 
- [Fine-tuning the ASR model - Hugging Face Audio Course](https://huggingface.co/learn/audio-course/en/chapter5/fine-tuning)
- [Exploration of Whisper fine-tuning strategies for low-resource ASR | EURASIP Journal on Audio, Speech, and Music Processing](https://asmp-eurasipjournals.springeropen.com/articles/10.1186/s13636-024-00349-3)

### PR DESCRIPTION
**Pull request by Kostis-S-Z**
# What's changing

Closes #9

### CODE REVIEW

**Comment by stefanfrench** [comment]
removing this to cut down the READ.me (I think we can add it back in the docs):

"This blueprint consists of three independent, yet complementary, components:

Transcription app: A simple UI that lets you record your voice, pick any HF STT/ASR model, and get an instant transcription.
Dataset maker app: Another UI app that enables you to easily and quickly create your own Speech-to-Text dataset.
Finetuning script: A script to finetune your own STT model, either using Common Voice data or your own local data created by the Dataset maker app."

**Comment by Kostis-S-Z** [comment]
The diagram looks great! Just some small comments:
> Initial transcription testing on ~base~ **baseline** ASR model
> Finetune an ASR model using your data ~and optionally~ _**OR**_ CommonVoice data

The phrase "and optionally" makes it seem as if you could do both at the same time with one script.

> ~Retest~ **Evaluate** transcription accuracy with finetuned ASR model

File: README.md [hunk 1]
```diff
@@ -14,33 +9,54 @@
...
+### 📖 For more detailed guidance on using this project, please visit our [Docs here](https://mozilla-ai.github.io/speech-to-text-finetune/)
+
 ### Built with
-- Python 3.10+
-- [Hugging Face](https://huggingface.co/)
-- [Gradio](https://www.gradio.app/)
-- [Common Voice](https://commonvoice.mozilla.org)
+![Python](https://img.shields.io/badge/Python-3.10%2B-blue) [![Hugging Face](https://img.shields.io/badge/Hugging%20Face-%F0%9F%A4%97-yellow)](https://huggingface.co/) [![Gradio](https://img.shields.io/badge/Gradio-%F0%9F%8E%A8-green)](https://www.gradio.app/) [![Common Voice](https://img.shields.io/badge/Common%20Voice-%F0%9F%8E%A4-orange)](https://commonvoice.mozilla.org)
+
+### 📖 For more detailed guidance on using this project, please visit our [Docs here](https://mozilla-ai.github.io/speech-to-text-finetune/)
```
Comment (stefanfrench) [inline README.md:45]: You've got "### 📖 For more detailed guidance on using this project, please visit our [Docs here](https://mozilla-ai.github.io/speech-to-text-finetune/)" twice.

**Review by stefanfrench — APPROVED** [review]
You've got the link to the guidance docs in the read.me twice. 

Otherwise approved :)

File: README.md (hunk 1 above)
Comment (Kostis-S-Z) [reply README.md:45]: woops, good catch, thanks!
